extensions/PhabBugz/lib/Feed.pm

```diff
@@ -427,6 +403,70 @@ sub process_revision_change {
       INFO("Requested #release-managers review of $stack_revision_phid.");
     }
 
+    # Comment the uplift form rendered on Bugzilla
+    # TODO add rendered form to storage on Phabricator side.
+    # TODO test
+    my $comment_data = {
+        'bug_id' => $revision->bug->id,
+        'is_markdown' => 1,
+    };
+    my $comment = Bugzilla::Comment->insert_create_data($comment_data);
+
+    # If manual QE is required, set the Bugzilla flag
+    # TODO test
+    if ($revision->uplift_form->{"Needs manual QE test"}) {
+        # TODO how to set bug flags?
+        # TODO set the `qe-verify` flag to `?` here.
+        my $new_flags = {
+            type_id  => $flagtype->id,
+            status   => '?',
+            setter   => $setter,
+            flagtype => $flagtype,
+        };
+
+        $revision->bug->set_flags($revision->bug->flags, $new_flags);
+        $revision->bug->update();
+    }
```

Comment:
I need to check the value of one of the keys in `uplift_form` and set `qe-verify` flag to `?` if it is `true`.